stdlib/transactions/src/main/ballerina/transactions/2pc_transaction.bal

```diff
@@ -80,7 +82,8 @@ type TwoPhaseCommitTransaction object {
                         ret = {message:OUTCOME_HAZARD};
                     }
                     NotifyResult => {
-                        boolean localAbortSuccessful = abortResourceManagers(self.transactionId, self.transactionBlockId);
+                        boolean localAbortSuccessful = abortResourceManagers(self.transactionId,
```

Comment:
I would rather break it like this:
boolean localAbortSuccessful = 
......................................abortResourceManagers(self.transactionId, self.transactionBlockId);